File: test/fixtures/matlab/get_with_env_var.m
%% Web Access using Data Import and Export API
params = {'type' 'distribution'};
baseURI = 'http://localhost:28139/v2/images';
uri = [baseURI '?' char(join(join(params, '='), '&'))];
options = weboptions(...
    'MediaType', 'application/json',...
    'HeaderFields', {'Authorization' ['Bearer ' getenv('DO_API_TOKEN')]}...
);
response = webread(uri, options);

%% HTTP Interface
import matlab.net.*
import matlab.net.http.*

params = {'type' 'distribution'};
header = [
    HeaderField('Content-Type', 'application/json')
    HeaderField('Authorization', ['Bearer ' getenv('DO_API_TOKEN')])
]';
uri = URI('http://localhost:28139/v2/images', QueryParameter(params'));
response = RequestMessage('get', header).send(uri.EncodedURI);

File: test/fixtures/elixir/get_with_env_var.ex
response = HTTPoison.get!(
  "http://localhost:28139/v2/images",
  [
    {"Content-Type", "application/json"},
    {"Authorization", "Bearer " <> System.get_env("DO_API_TOKEN", "")}
  ],
  [
    params: [
      {"type", "distribution"}
    ]
  ]
)
